Codeinwp/feedzy-rss-feeds
# fix: filter key for select components

Closes https://github.com/Codeinwp/feedzy-rss-feeds-pro/issues/684

## Summary

~This is an odd problem. When you change the values of the select component and then add some filter keywords in the input field, the last modified select component loses its values (sometimes happens to both) .~

~I could **NOT** find the cause of this problem (I could not determine if it is a problem with Chosen lib or Tagify one.). This does not happen to other select components of the page.~

~As a temporary fix, I am saving the values in the session storage and re-apply them before sending the form to the server.~

Fixed the input reset query when a new tag is added.

## Testing

1. Create a new Import Feed;
2. On the Filter tab, set Author on both Select Component;
3. Add some keywords in the input fields;
4. Make some changes to other options, then Save;
5. Re-enter the feed settings;
6. You should see the Author set for both Select Components.


![image](https://github.com/Codeinwp/feedzy-rss-feeds/assets/17597852/c2227b2e-f87d-445e-aa37-8f08f50afbea)
